sdk/cognitiveservices/cognitiveservices-luis-authoring/package.json

```diff
@@ -57,9 +50,7 @@
   "scripts": {
     "build": "tsc && rollup -c rollup.config.js && npm run minify",
     "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/cognitiveservices-luis-authoring.js.map'\" -o ./dist/cognitiveservices-luis-authoring.min.js ./dist/cognitiveservices-luis-authoring.js",
-    "prepack": "npm install && npm run build",
-    "test": "ts-mocha -p tsconfig.test.json test/**/*.test.ts --timeout 100000"
```

Comment:
This test command was manually added to run the tests in the `cognitiveservices-luis-authoring/tests` folder. Please retain it and ensure all the tests are successful